OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -85,14 +92,18 @@ public ClipboardTechSupport(TinyIoCContainer container)
                 // TODO: plugin.pluginVersion has FileVersion, ProductVersion, etc, but all as a string.
                 // For now, ask FileVersionInfo to get this for us.
                 var fullPath = plugin.pluginFile.FullName;
+                var fullpathregex = Regex.Replace(fullPath, ACTPathPattern, ACTPathReplace);
+                fullpathregex = fullpathregex.Replace(Environment.UserName, "<USER>");
```

Comment:
```suggestion
                var censoredFullPath = Regex.Replace(fullPath, ACTPathPattern, ACTPathReplace);
                censoredFullPath = censoredFullPath.Replace(Environment.UserName, "<USER>");
```

Minor nitpick: Can you rename the variable to make it clearer what it contains? The same goes `UrlRegex` below. After that, I'd say this is ready to merge.